testing/services/dummy/dsubscriber/README.md

```diff
@@ -0,0 +1,16 @@
+# Site Registry
```

Comment:
Can we remove this or update it? It's an old Markdown document for the registry site.